cves/kernel/CVE-2016-8650.yml

```diff
@@ -305,23 +316,27 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
+  discussed_as_security: true
+  any_discussion: true
+  note: |
```

Comment:
Capitalize your statement and add a period at the end. You could separate this into two sentences for improved readability.